onnxmltools/convert/common/_topology.py

```diff
@@ -581,11 +595,13 @@ def _prune(self):
             for onnx_name in abandoned_variable_names:
                 scope.delete_local_variable(onnx_name)
 
-    def compile(self):
+    def compile(self, custom_conversions):
         '''
         This function aims at giving every operator enough information so that all operator conversions can happen
         independently. We also want to check, fix, and simplify the network structure here.
+        :param custom_conversions: the dictionary of the user customized conversion functions.
         '''
+        self.custom_conversions = custom_conversions
```

Comment:
If it's a field of a topology, please assign it a default value in `Topology` constructor.